charts/seed-operatingsystemconfig/original/templates/scripts/_init-containerd.sh

```diff
@@ -12,4 +12,16 @@
           mkdir -p /etc/containerd
           containerd config default > "$FILE"
         fi
+
+        BIN_PATH={{ .Values.osc.cri.containerRuntimesBinaryPath }}
+        mkdir -p $BIN_PATH
+
+        ENV_FILE=/etc/systemd/system/containerd.service.d/30-env_config.conf
```

Comment:
Hi no because the file is generated with a script to concat the $PATH variable. Each OS has a different PATH and i didn't find a way to concat the PATH in a static drop in file.

Reply:
ok got it thanks!